packages/client/test/end-to-end/ProxyPublishSubscribe.test.ts

```diff
@@ -61,8 +61,8 @@ describe('PubSub with proxy connections', () => {
         const proxyUser1 = await proxyClient1.getAddress()
         const proxyUser2 = await proxyClient2.getAddress()
 
-        proxyPeerKey1 = await proxyClient1.getNodeId()
-        proxyPeerKey2 = await proxyClient2.getNodeId()
+        proxyPeerKey1 = await proxyClient1.getNodeId() as NodeID
+        proxyPeerKey2 = await proxyClient2.getNodeId() as NodeID
```

Comment:
Shouldn't the result of `getNodeId()` already be a `NodeID`, i.e. why re-cast here?

Reply:
Maybe `Client#getNodeId` should return `NodeID` if it isn't already.

Reply:
Changed the return type